[CI] Several test timeouts in 6.8.16 mixedClusterTests

This doesn't seem to be a specific test failure but more of a cluster of timouts when running MixedClusterClientYamlTestSuiteIT.
I'm opening this one for a particular failure on 7.13 but suspect an underlying issue.

**Build scan**:

https://gradle-enterprise.elastic.co/s/otc2tm2y4vuuc

**Repro line**:

```
:qa:mixed-cluster:v6.8.16#mixedClusterTest' \
  -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" \
  -Dtests.method="test {p0=search/160_exists_query/Test exists query on long field in empty index}" \
  -Dtests.seed=D055913F7CE762 \
  -Dtests.bwc=true \
  -Dtests.locale=ar-JO \
  -Dtests.timezone=America/Belize \
  -Druntime.java=8
```

**Reproduces locally?**:

No

**Applicable branches**:

Seen something similar on several 7.x branches on various rest test

**Failure history**:

**Failure excerpt**:

Test timeouts, here's a direct link to the jstack output: https://gradle-enterprise.elastic.co/s/otc2tm2y4vuuc/tests/:qa:mixed-cluster:v6.8.16%23mixedClusterTest

I also saw this in the logs which might show why cluster formation is broken:

```
 Caused by: org.elasticsearch.action.search.VersionMismatchException: One of the shards is incompatible with the required minimum version [7.13.0]
»  	at org.elasticsearch.action.search.AbstractSearchAsyncAction.getConnection(AbstractSearchAsyncAction.java:704) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.action.search.AbstractSearchAsyncAction.lambda$checkMinimumVersion$0(AbstractSearchAsyncAction.java:243) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90) ~[?:1.8.0_281]
»  	at java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1361) ~[?:1.8.0_281]
»  	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126) ~[?:1.8.0_281]
»  	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:499) ~[?:1.8.0_281]
»  	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:486) ~[?:1.8.0_281]
»  	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472) ~[?:1.8.0_281]
»  	at java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:230) ~[?:1.8.0_281]
»  	at java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:196) ~[?:1.8.0_281]
»  	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_281]
»  	at java.util.stream.ReferencePipeline.anyMatch(ReferencePipeline.java:449) ~[?:1.8.0_281]
»  	at org.elasticsearch.action.search.AbstractSearchAsyncAction.checkMinimumVersion(AbstractSearchAsyncAction.java:242) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.action.search.AbstractSearchAsyncAction.run(AbstractSearchAsyncAction.java:216) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executePhase(AbstractSearchAsyncAction.java:424) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	... 78 more
```

@jimczi looking closer in the node logs (eg in https://gradle-enterprise.elastic.co/s/ufgbyhkw3hgsi), I've seen the following traces that could point to a real-non-timeout problem:
```
» [2021-04-29T13:24:36,992][DEBUG][o.e.a.s.TransportSearchAction] [v6.8.16-3] All shards failed for phase: [query]
»  org.elasticsearch.ElasticsearchException$1: [composite] aggregation cannot be used with a parent aggregation of type: [TermsAggregatorFactory]
»  	at org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:657) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executeNextPhase(AbstractSearchAsyncAction.java:131) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.search.AbstractSearchAsyncAction.onPhaseDone(AbstractSearchAsyncAction.java:259) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.search.InitialSearchPhase.onShardFailure(InitialSearchPhase.java:100) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.search.InitialSearchPhase.access$100(InitialSearchPhase.java:48) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.search.InitialSearchPhase$2.lambda$onFailure$1(InitialSearchPhase.java:220) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.search.InitialSearchPhase.maybeFork(InitialSearchPhase.java:174) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.search.InitialSearchPhase.access$000(InitialSearchPhase.java:48) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.search.InitialSearchPhase$2.onFailure(InitialSearchPhase.java:220) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.search.SearchExecutionStatsCollector.onFailure(SearchExecutionStatsCollector.java:73) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.ActionListenerResponseHandler.handleException(ActionListenerResponseHandler.java:59) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.search.SearchTransportService$ConnectionCountingHandler.handleException(SearchTransportService.java:463) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleException(TransportService.java:1126) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.transport.TcpTransport.lambda$handleException$24(TcpTransport.java:1011) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.EsExecutors$DirectExecutorService.execute(EsExecutors.java:193) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.transport.TcpTransport.handleException(TcpTransport.java:1009) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.transport.TcpTransport.handlerResponseError(TcpTransport.java:1001) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.transport.TcpTransport.messageReceived(TcpTransport.java:950) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.transport.TcpTransport.inboundMessage(TcpTransport.java:763) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.transport.netty4.Netty4MessageChannelHandler.channelRead(Netty4MessageChannelHandler.java:53) [transport-netty4-client-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:323) [netty-codec-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:297) [netty-codec-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.handler.logging.LoggingHandler.channelRead(LoggingHandler.java:241) [netty-handler-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1434) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:965) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:656) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:556) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:510) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:470) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
»  	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:909) [netty-common-4.1.32.Final.jar:4.1.32.Final]
»  	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
»  Caused by: java.lang.IllegalArgumentException: [composite] aggregation cannot be used with a parent aggregation of type: [TermsAggregatorFactory]
»  	at org.elasticsearch.search.aggregations.bucket.composite.CompositeAggregationBuilder.doBuild(CompositeAggregationBuilder.java:176) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:139) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:335) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.aggregations.AggregatorFactory.<init>(AggregatorFactory.java:187) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.aggregations.support.ValuesSourceAggregatorFactory.<init>(ValuesSourceAggregatorFactory.java:40) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.aggregations.bucket.terms.TermsAggregatorFactory.<init>(TermsAggregatorFactory.java:73) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.aggregations.bucket.terms.TermsAggregationBuilder.innerBuild(TermsAggregationBuilder.java:338) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:311) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:37) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:139) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:335) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.parseSource(SearchService.java:848) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.createContext(SearchService.java:653) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.createAndPutContext(SearchService.java:604) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:388) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.access$100(SearchService.java:127) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService$2.onResponse(SearchService.java:360) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService$2.onResponse(SearchService.java:356) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService$4.doRun(SearchService.java:1129) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:41) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:778) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_242]
»  	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_242]
```
```
»  org.elasticsearch.transport.RemoteTransportException: [v6.8.16-3][127.0.0.1:63975][indices:data/read/search[phase/query]]
»  Caused by: org.elasticsearch.search.query.QueryPhaseExecutionException: Rescore window [10001] is too large. It must be less than [10000]. This prevents allocating massive heaps for storing the results to be rescored. This limit can be set by changing the [index.max_rescore_window] index level setting.
»  	at org.elasticsearch.search.DefaultSearchContext.preProcess(DefaultSearchContext.java:229) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.query.QueryPhase.preProcess(QueryPhase.java:91) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.createContext(SearchService.java:665) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.createAndPutContext(SearchService.java:604) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:388) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.access$100(SearchService.java:127) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService$2.onResponse(SearchService.java:360) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService$2.onResponse(SearchService.java:356) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService$4.doRun(SearchService.java:1129) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:41) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:778) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_242]
»  	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_242]
»  	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
```
```
»  org.elasticsearch.transport.RemoteTransportException: [v6.8.16-1][127.0.0.1:64633][indices:data/read/search[phase/query]]
»  Caused by: org.elasticsearch.search.SearchException: cannot use `collapse` in conjunction with `rescore`
»  	at org.elasticsearch.search.SearchService.parseSource(SearchService.java:1105) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.createContext(SearchService.java:753) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:419) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.search.SearchService.lambda$executeQueryPhase$2(SearchService.java:394) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.ActionRunnable.lambda$supply$0(ActionRunnable.java:47) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:62) ~[?:?]
»  	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:732) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26) ~[elasticsearch-6.8.16-SNAPSHOT.jar:6.8.16-SNAPSHOT]
»  	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_242]
»  	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_242]
»  	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
```

Can you please take a look?

I'm going to mute these because they're getting very noisy.

--- a/buildSrc/src/main/java/org/elasticsearch/gradle/internal/ElasticsearchTestBasePlugin.java
+++ b/buildSrc/src/main/java/org/elasticsearch/gradle/internal/ElasticsearchTestBasePlugin.java
@@ -176,7 +176,7 @@ public class ElasticsearchTestBasePlugin implements Plugin<Project> {
 
             if (OS.current().equals(OS.WINDOWS) && System.getProperty("tests.timeoutSuite") == null) {
                 // override the suite timeout to 30 mins for windows, because it has the most inefficient filesystem known to man
-                test.systemProperty("tests.timeoutSuite", "1800000!");
+                test.systemProperty("tests.timeoutSuite", "2400000!");
             }
 
             /*

